### PR TITLE
Fixes #462 Add Video search support for Ask 

### LINKED ISSUE #462: Add Video search support for Ask
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

Currently, no support for Ask Video search.
**Expected behavior:**

Add Video search support for Ask
**Steps to reproduce:**
**yes, wanna work on this issue**

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #462 
Addresses #320 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added Video search support for Ask
-

Providing the heroku deployment at https://salty-spire-70237.herokuapp.com/
@mariobehling @ParthS007 @vaibhavsingh97 Please review. Thanks !!

### CODE REVIEW

**Comment by bhaveshAn** [comment]
Added tests at `test/tests_ask.py` .
@cclauss @ParthS007 @vaibhavsingh97 @gabru-md Please review.

**Comment by bhaveshAn** [comment]
@vaibhavsingh97 Please review asap...

**Comment by mariobehling** [comment]
Please resolve conflicts

File: app/scrapers/__init__.py [hunk 1]
```diff
@@ -42,6 +42,8 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
+    elif engine in ('ask',) and qtype == 'vid':
```
Comment (gabru-md) [inline app/scrapers/__init__.py]: Why use `in ('ask',)`
You can go with `if engine == 'ask'`
since there is only one option to choose from?

Comment (bhaveshAn) [reply app/scrapers/__init__.py]: So that other engines can also be added in this tuple, if there implementation is same.

**Pull request merged**